nixos/modules/system/boot/systemd-nspawn.nix

```diff
@@ -42,8 +42,127 @@ let
     (assertValueOneOf "VirtualEthernet" boolValues)
   ];
 
+  bindMounts = mkOption {
+    type = with types; attrsOf (submodule {
+      options = {
+        source = mkOption {
+          type = either str path;
```

Comment:
The path will be copied to the Nix-Store and the store-path will be mounted when using `types.path` here. We either want to drop it completely or do something like `apply = toString` (`"${./.}"` is a store-path, `"${toString ./.}"` is the (unvalidated) string-representation of the path-value)

Reply:
Thanks, dropped it for simplicity.

> The path will be copied to the Nix-Store and the store-path will be mounted when using `types.path` here.

A bit surprising for me.